pkgs/applications/science/misc/golly/default.nix

```diff
@@ -35,24 +36,45 @@ stdenv.mkDerivation rec {
     wrapGAppsHook3
   ];
 
+  enableParallelBuilding = true;
+
+  preConfigure = lib.optionalString stdenv.isDarwin ''
+    export NIX_CFLAGS_COMPILE+=" -L$SDKROOT/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries"
```

Comment:
You can replace `-lGL -lGLU` with this in the relevant makefile instead (seems more elegant)